queries/nix.scm

```diff
@@ -0,0 +1,146 @@
+;; Statements
+
+[
+    (binding)
+] @statement
+;; This matches assert statements as stand-alone statements. This
+;; is because it's common to have a series of assert statements in a file, which
+;; would otherwise all match as a single statement. See the playground/nix/asserts.nix
+;; file for an example.
+(assert_expression
+    "assert" @statement.start
+    condition: (_)
```

Comment:
not sure you need this line

Reply:
You are right, I didn't for the assert statement match, however as I was about to remove it I realized I could add `@condition` there which is really nice, so will keep it.